src/main/java/com/spartronics4915/frc2019/paths/TrajectoryGenerator.java

```diff
@@ -104,18 +104,30 @@ public MirroredTrajectory(Trajectory<TimedState<Pose2dWithCurvature>> right)
             public final Trajectory<TimedState<Pose2dWithCurvature>> right;
         }
 
-        public final MirroredTrajectory test;
+        public final MirroredTrajectory straightTest;
+        public final MirroredTrajectory curvedTest;
```

Comment:
just a nit wrt 254 naming. MirroredTrajectory would be better named MirrorableTrajectory.